From 14d8ca115cca04b29c8b0cfcdda42197e000fdd8 Mon Sep 17 00:00:00 2001
Message-ID: <14d8ca115cca04b29c8b0cfcdda42197e000fdd8.1704423368.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1704423368.git.sharlatanus@gmail.com>
References: <cover.1704423368.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 5 Jan 2024 02:39:54 +0000
Subject: [PATCH 1/3] gnu: go-github-com-hjson-hjson-go: Fix commit.

This package was updated in ada2152893eda2301890e6a438c3e9fb4e0f190d and
moved with lower version which broke yggdrasil.

Author:     Artyom V. Poptsov <poptsov.artyom@gmail.com>
AuthorDate: Wed Nov 1 00:15:16 2023 +0300
Commit:     Christopher Baines <mail@cbaines.net>
CommitDate: Fri Nov 3 10:14:06 2023 +0000

gnu: go-github-com-hjson-hjson-go: Update to 4.3.1.

* gnu/packages/golang.scm (go-github-com-hjson-hjson-go): Update to 4.3.1.

Change-Id: Iee2aef614686cbf79f388e2dd9435745d5502e2a
---
 gnu/packages/golang-web.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 7759916c35..3e60bfaf33 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -21,6 +21,7 @@
 ;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
 ;;; Copyright © 2023 Katherine Cox-Buday <cox.katherine.e@gmail.com>
 ;;; Copyright © 2023 Nicolas Graves <ngraves@ngraves.fr>
+;;; Copyright © 2023 Artyom V. Poptsov <poptsov.artyom@gmail.com>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -870,7 +871,7 @@ (define-public go-github-com-gorilla-websocket
 (define-public go-github-com-hjson-hjson-go
   (package
     (name "go-github-com-hjson-hjson-go")
-    (version "3.1.0")
+    (version "4.3.1")
     (source
      (origin
        (method git-fetch)
@@ -879,7 +880,7 @@ (define-public go-github-com-hjson-hjson-go
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "1dfdiahimg6z9idg8jiqxwnlwjnmasbjccx8gnag49cz4yfqskaz"))))
+        (base32 "138vmbnrwzxf64cia27k407clrydvs2jx927dlv6ziydiqyvy7m3"))))
     (build-system go-build-system)
     (arguments
      '(#:import-path "github.com/hjson/hjson-go"))
-- 
2.41.0

